src/SFA.DAS.IdentifyDataLocks.IntegrationTests/LearnerWithStoppedApprenticeship/LearnerWithStoppedApprenticeship.cs

```diff
@@ -17,4 +20,102 @@ public async Task RendersDataMismatches()
             result.Content.Headers.ContentType.ToString().Should().Be("text/html; charset=utf-8");
         }
     }
+
+    [Explicit]
+    public class LearnerWithMultiplePriceEpisodes : WebApplicationTestFixture
+    {
+        [Test]
+        public async Task RendersTNPForOncePriceEpisodes()
+        {
+            await Arrange("SFA.DAS.IdentifyDataLocks.IntegrationTests.TestData");
+
+            var result = await CreateClient().GetAsync("/Learner/2839925663");
+
+            result.EnsureSuccessStatusCode(); // Status Code 200-299
+            result.Content.Should().NotBeNull();
+            var body = await result.Content.ReadAsStringAsync();
+
+            var parser = new AngleSharp.Html.Parser.HtmlParser();
+            var document = parser.ParseDocument(body);
+            var tnp1Row = document.QuerySelector("tr.tnp1");
+            tnp1Row.Should().NotBeNull();
+            tnp1Row.Children.Should().BeEquivalentTo(
+                new[]
+                {
+                    new { TextContent = "TNP 1" },
+                    new { TextContent = "n/a" },
+                    new { TextContent = "Date: 01/12/2019\n        Price: �24,872.00" },
```

Comment:
Well done on this, but the white space and new line characters make these tests seem brittle.

Reply:
Yes, I agree.  It's caused by the indentation in the .cshtml file.  I haven't come up with a way to deal with it yet.